tools/rosmsg/src/rosmsg/__init__.py

```diff
@@ -693,9 +701,14 @@ def rosmsg_cmd_list(mode, full, argv=None):
         raise ValueError('Unknown mode for iterate_packages: %s'%mode)
     rospack = rospkg.RosPack()
     packs = sorted([x for x in iterate_packages(rospack, mode)])
+    if (package_name is not None) and (package_name not in zip(*packs)[0]):
```

Comment:
The first condition could be simplied to: `not package_name`

The parenthesis around the conditions are not necessary.

Same four lines below.